gettsim/interface.py

```diff
@@ -815,7 +836,14 @@ def _create_aggregation_functions(user_and_internal_functions, targets, data_col
         agg_col: {"aggr": "sum", "source_col": rchop(rchop(agg_col, "_tu"), "_hh")}
         for agg_col in automated_sum_aggregation_cols
     }
-    aggregation_dict = {**aggregation_dict, **automated_sum_aggregation_specs}
+
+    # Add automated aggregation specs.
+    # Note: For duplicate keys, automated specs are treated with lower priority.
```

Comment:
```suggestion
    # Note: For duplicate keys, explicitly set specs are treated with higher priority than automated specs.
```
Better the positive version. More importantly, keep it the same way as next docstring.